governance/polls/Vault Compensation Denomination - April 13, 2020.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Vault Compensation Denomination Poll - April 13, 2020
+summary: Signal whether to provide compensation to vault holders in the vault's token (ETH/BAT) or in DAI
+discussion_link: https://forum.makerdao.com/t/1822
+poll_rules: The voter may select to vote for one of the poll options or they may elect to abstain from the poll entirely
+options:
+   0: Abstain
+   1: ETH or BAT (the asset that was liquidated) 
+   2: DAI
+
+---
+# Vault Compensation Denomination Poll - April 13, 2020
+
+The Maker Foundation Interim Governance Facilitator has placed a Governance Poll into the [voting system](https://vote.makerdao.com/polling) on behalf of the community which the community can use to signal whether to move ahead with a plan to provide compensation to vault holders.
+
+This Governance Poll ([FAQ](https://community-development.makerdao.com/makerdao-scd-faqs/scd-faqs/governance)) will be active for 7 days beginning on Monday, April 13 at 4 PM UTC.
+
+## Review
+
+[An initial poll on whether to compensate vault holders has passed.](https://vote.makerdao.com/polling-proposal/qmwfvvguaf8rz8xwgv2cqnzzt9t5h6epzh17qmk2ue99y4)
+
```

Comment:
Many voters may only want the tl/dr and would be well served to be presented with the main arguments up front? 
This could also provide a disambiguation for the substance/consequences/nuances of the two options which may be more subtle as mentioned in the comments for the summary in line 3.
More detail/technical explanations can follow.
```suggestion

This poll follows from the initial poll to offer the choice in the nature of said compensation namely (ETH/BAT) or DAI.
An overview of the main arguments for and against:
Arguments for compensation in ETH:
Arguments against compensation in ETH:
Arguments for compensation in DAI:
Arguments against compensation in DAI:
```

Reply:
Dunno, I think the poll is already simple enough. You might try to make a complete summary and then see whether the summary is actually shorter than the poll or just increases the length.

Reply:
It could be an addition so long as we don't duplicate. The problem with not presenting a summary like this is that the question at hand could become distorted. Line 22, for example asks "do we accept this liability". A voter may think “no I don’t want to take on additional liability” and vote for compensation in DAI, but the question framed that way may miss some context and probable consequences. The same voter could feel it was money wasted should DAI compensation be made and Maker still “receive bad press and acrimonious gestures” and “users not be happy”. The benefits of compensation (the agreed outcome) would not have been achieved even though cost was incurred. 
A pro/con presentation should avoid this as all main arguments would be stated in one place. I'll take a try populating the arguments if that will help - just didn't want to add it to one pull suggestion

Reply:
Also realise now we could just have "arguments for" for either and make them positive/negative statements if need be. Should be simpler

Reply:
Edits to for/against arguments are needed, but I think this gets to the heart of it?

```suggestion

This poll follows from the previous and offers the choice in the nature of said compensation namely (ETH/BAT) or DAI.
Main arguments for compensation in ETH: 
- ETH compensation pegs compensation to the asset that vault owners would have had the protocol not performed in a sub-optimal manner. Therefore it avoids the risk that compensation occurs, but users feel cheated and the objectives of compensation (happy users, no distortion of symmetry, positive impact on reputation and future mass adoption) still not be achieved.
Main arguments for compensation in DAI: 
- Compensation in DAI would be a lower compensation value since the value of ETH in DAI terms has appreciated after the liquidation event.

```

Reply:
I like where you're going with this, but give me a sec to tweak the language.